environments/edusources/acceptance/harvester/invoke.yml

```diff
@@ -12,7 +12,7 @@ django:
     keep_versions: 3
   repositories:
     edurep: "https://wszoeken.edurep.kennisnet.nl"
-    sharekit: "https://api.acc.surfsharekit.nl"
+    sharekit: "https://api.surfsharekit.nl"
```

Comment:
dit moet zo blijven?

Reply:
Het zal steeds wisselen. Wanneer Martine een demo gaat geven wilt ze graag met acc verbinden. Maar acc is nu stuk, dus dan is een productie koppeling handiger.